### PR TITLE
Document that 7.6 Operator is compatible with 7.5 Keycloak

### LINKED ISSUE #1583: Document that 7.6 Operator is compatible with 7.5 Keycloak
### Description

Document that 7.6 Operator is compatible with 7.5 Keycloak

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by vmuzikar**
Closes #1583

### CODE REVIEW

File: server_installation/topics/operator/installation.adoc [hunk 1]
```diff
@@ -60,6 +60,29 @@ However, if you want to start tracking all Operator activities before creating c
 endif::[]
 
 * For more information on OpenShift Operators, see the link:https://docs.openshift.com/container-platform/4.4/operators/olm-what-operators-are.html[OpenShift Operators guide].
+
+ifeval::[{project_community}==false]
+[[_operator_rhsso_compatibility]]
+=== Operator compatibility with different RH-SSO versions
+The RH-SSO Operator 7.6 is fully compatible with RH-SSO 7.5. The 7.6 Operator by default deploys RH-SSO 7.6. To change this
+behaviour and use the RH-SSO 7.5 image you need to override the image coordinates by setting the `RELATED_IMAGE_RHSSO`
+environment variable in the Operator pod. For that, please update your OLM `Subscription` for the RH-SSO Operator inside
```
Comment (andymunro) [inline server_installation/topics/operator/installation.adoc]:
```suggestion
=== Operator compatibility with different RH-SSO versions
The RH-SSO Operator 7.6 is fully compatible with RH-SSO 7.5. The 7.6 Operator by default deploys RH-SSO 7.6. To use the RH-SSO 7.5 image instead, override the image coordinates by setting the `RELATED_IMAGE_RHSSO`
environment variable in the Operator pod. To make that change, update your OLM `Subscription` for the RH-SSO Operator inside
```

**Comment by vmuzikar** [comment]
@andymunro Thanks for the review!

File: server_installation/topics/operator/installation.adoc [hunk 2]
```diff
@@ -152,6 +174,7 @@ However, if you want to start tracking all Operator activities before creating c
 endif::[]
 
 * For more information on OpenShift Operators, see the link:https://docs.openshift.com/container-platform/4.4/operators/olm-what-operators-are.html[OpenShift Operators guide].
+endif::[]
```
Comment (ahus1) [inline server_installation/topics/operator/installation.adoc]: I saw that the build was broken due to internal anchors not working. It seems to me that this `endif::[]` should be in line 85 just before the next `ifeval::[]` starts.

Comment (vmuzikar) [reply server_installation/topics/operator/installation.adoc]: A very good catch, thank you! You are of course correct. Should be fixed now.

**Comment by andymunro** [comment]
@vmuzikar Is there a backport Jira for this change so it will go in when we publish 7.6 on Thursday?

**Comment by vmuzikar** [comment]
@andymunro There is [this one](https://issues.redhat.com/browse/RHSSO-2076).

**Comment by andymunro** [comment]
Okay, thanks. Actually, I mean have you created an MR for the 7.6 Jira that you mention.  We will need that MR merged by tomorrow morning.

**Review by andymunro — APPROVED** [review]

**Review by miquelsi — APPROVED** [review]

**Pull request merged**